client/src/App.js

```diff
@@ -8,18 +8,19 @@ import SignIn from "./pages/SignIn";
 import SignUp from "./pages/SignUp";
 import PrivateRoute from "./utils/routes";
 import CardModal from "./components/CreateCard/Modal";
+import { loggedIn } from "./AuthService";
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <PrivateRoute path="/dashboards" component={DashBoard} />
-        <Route path="/signin" component={SignIn} />
-        <Route path="/signup" component={SignUp} />
+        <Route path='/signin' component={SignIn} />
+        <Route path='/signup' component={SignUp} />
+        <PrivateRoute loggedIn={loggedIn()} path='/dashboards/:dashboardId' component={DashBoard} />
         <Route
```

Comment:
no need to pass in loggedin, this logic is already handled in private route component